src/pages/students/[student_id].tsx

```diff
@@ -1,44 +1,85 @@
 import { trpc } from "@/client/lib/trpc";
+import EditStudentRow from "@/components/editStudentRow/EditStudentRow";
 import $button from "@/styles/Button.module.css";
 import $home from "@/styles/Home.module.css";
 import $input from "@/styles/Input.module.css";
 import Box from "@mui/material/Box";
 import Button from "@mui/material/Button";
 import Container from "@mui/material/Container";
 import Modal from "@mui/material/Modal";
+import Stack from "@mui/material/Stack";
+import { addYears, format, parseISO, subDays } from "date-fns";
 import Image from "next/image";
-import Link from "next/link";
 import { useRouter } from "next/router";
 import { useState } from "react";
 import Iep from "../../components/iep/Iep";
 import noGoals from "../../public/img/no-goals-icon.png";
 import $Form from "../../styles/Form.module.css";
+import $Image from "../../styles/Image.module.css";
 import $Modal from "../../styles/Modal.module.css";
 import $StudentPage from "../../styles/StudentPage.module.css";
-import $Image from "../../styles/Image.module.css";
-import { parseISO, addYears, subDays, format } from "date-fns";
 
 const ViewStudentPage = () => {
   const [createIepModal, setCreateIepModal] = useState(false);
   const [archivePrompt, setArchivePrompt] = useState(false);
   const [startDate, setStartDate] = useState("");
   const [endDate, setEndDate] = useState("");
+  const [viewState, setViewState] = useState(0);
```

Comment:
if there's only two states, could this be

```suggestion
  const [isEditing, setIsEditing] = useState(false);
```

instead?

Reply:
I have not implemented this yet. I like the viewState way, but I will change it to boolean if you think it would help performance.

Reply:
my personal preference is a boolean as it feels a bit clearer to me, but it's fine to leave as-is too :), no performance impact

Reply:
Okay great. I will leave it for now but may change it later. I am imagining there may be something similar we would have to do for ticket #200 and at that point, I would change both if they are both just using two states, which would be cleaner with a boolean.